src/client/Client.cc

```diff
@@ -9147,7 +9147,7 @@ int Client::_write(Fh *f, int64_t offset, uint64_t size, const char *buf,
   utime_t lat;
   uint64_t totalwritten;
   int have;
-  int r = get_caps(in, CEPH_CAP_FILE_WR|CEPH_CAP_AUTH_SHARED,
+  int64_t r = get_caps(in, CEPH_CAP_FILE_WR|CEPH_CAP_AUTH_SHARED,
```

Comment:
get_caps returns a real int, not an int64.

Reply:
Oops, that one is probably superfluous. I'll remove that delta. Good catch.